guac-restAPI-integration.md

```diff
@@ -0,0 +1,145 @@
+# GUAC REST API Documentation
+
+## Overview
+
+The GUAC REST API provides an alternative to the GUAC GraphQL API, offering simplified access to advanced queries. It is ideal for users or developers who prefer REST endpoints over GraphQL queries and need to integrate GUAC services into their own tools and scripts.
+
+## Base URL
+```
+http://<host>:<port>/
+```
+
+## Endpoints
+
+Here’s your `.yml` converted into markdown following your previous example:
```

Comment:
I assume this is an artifact of using a generative AI tool to perform the conversion? That's fine, but let's make sure to remove this line. :-)